test/sol6/reserves/kyberFprReserveV2.js

```diff
@@ -0,0 +1,3060 @@
+const MockConversionRates = artifacts.require("MockConversionRates");
+const TestToken = artifacts.require("TestToken");
+const WethToken = artifacts.require("WethToken");
+const Reserve = artifacts.require("KyberFprReserveV2");
+const MockSanityRates = artifacts.require("MockSanityRates");
+const NoPayableFallback = artifacts.require("NoPayableFallback");
+
+const Helper = require("../../helper.js");
+const reserveSetup = require("../../reserveSetup.js");
+const BN = web3.utils.BN;
+
+//global variables
+//////////////////
+const {precisionUnits, ethDecimals, ethAddress, zeroAddress, zeroBN, MAX_ALLOWANCE} = require("../../helper.js");
+const {expectEvent, expectRevert} = require('@openzeppelin/test-helpers');
+
+//balances
+let expectedReserveBalanceWei = new BN(0);
+let expectedReserveBalanceWeth = new BN(0);
+let reserveTokenBalance = [];
+let reserveTokenImbalance = [];
+
+//permission groups
+let admin;
+let operator;
+let alerter;
+let network;
+let withdrawAddress;
+
+//contracts
+let weth;
+let convRatesInst;
+let reserveInst;
+
+//block data
+let currentBlock;
+let maxGasPrice = new BN(150).mul(new BN(10).pow(new BN(9))); // 150 * 10^9
+
+//tokens data
+////////////
+let numTokens = 5;
+let tokens = [];
+let tokenDecimals = [];
+let tokenAdd = [];
+
+// all price steps in bps (basic price steps).
```

Comment:
I think this comment doesn't belong here.

its good to have it in other places.
like dev portal or conversion rate test file.

Reply:
ok will remove from reserve test file